opt_einsum/helpers.py

```diff
@@ -167,3 +168,95 @@ def flop_count(idx_contraction, inner, num_terms, size_dictionary):
         op_factor += 1
 
     return overall_size * op_factor
+
+
+def rand_equation(n, reg, n_outer, dmin=2, dmax=9, seed=None):
```

Comment:
There might be a slight mismatch between styles here `dmin` `nouter` vs `d_min`, `n_outer`. I think your current nomenclature is the lesser of the evils, but worth thinking about.